app/css/index.css

```diff
@@ -676,72 +688,76 @@ video {
 	border-color: rgb(255 255 255 / var(--tw-border-opacity));
 }
 
-.bg-background {
-	--tw-bg-opacity: 1;
-	background-color: rgb(22 22 29 / var(--tw-bg-opacity));
-}
-
 .bg-accent {
 	--tw-bg-opacity: 1;
 	background-color: rgb(108 207 246 / var(--tw-bg-opacity));
 }
 
-.bg-white {
+.bg-background {
 	--tw-bg-opacity: 1;
-	background-color: rgb(255 255 255 / var(--tw-bg-opacity));
-}
-
-.bg-success\/10 {
-	background-color: rgb(152 206 0 / 0.1);
+	background-color: rgb(22 22 29 / var(--tw-bg-opacity));
 }
 
-.bg-error\/10 {
-	background-color: rgb(219 84 97 / 0.1);
+.bg-black\/20 {
```

Comment:
What is with the escaped `/` here?  Seems like it would be easier to just use a `-` or something that doesn't require escaping.